docs/words.txt

```diff
@@ -1,17 +1,28 @@
 aggregator
 app
+backend
 config
 Config
 configurables
 Cookiecutter
+crypto
 Einhorn
+gevent
 Gevent
+healthcheck
 hostname
 INI
 lifecycle
 namespace
+predecided
 pubsub
+Queing
```

Comment:
unless this is a proper noun, I believe the spelling is "queuing". 

question for learning: what's this file do?